### ISSUE TITLE
CONSTRUCT query with BIND or VALUES does not work as expected on a Solid pod

### ISSUE THREAD

**Issue by karelklima** [comunica/comunica#1327]
#### Issue type:

- :bug: Bug 

____
#### Description:

A CONSTRUCT query that binds variables in the WHERE clause returns no results. I am querying my user card on a Solid pod, trying to get to the following triple: `<https://karelklima.solidcommunity.net/profile/card#me> <http://xmlns.com/foaf/0.1/name> "Karel Klima"`

SPARQL query that works:
```
CONSTRUCT {
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
WHERE {
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
```

SPARQL query that returns no quads:
```
CONSTRUCT {
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
WHERE {
  BIND(<https://karelklima.solidcommunity.net/profile/card#me> AS ?iri)
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
```

SPARQL query that returns no quads either:
```
CONSTRUCT {
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
WHERE {
  VALUES ?iri {
    <https://karelklima.solidcommunity.net/profile/card#me>
  }
  ?iri <http://xmlns.com/foaf/0.1/name> ?iri_0 .
}
```

Gist with the complete code I am running: https://gist.github.com/karelklima/e061b730f6b1606960d448df99b4a972

Possibly related issue: https://github.com/comunica/comunica/issues/1201

Note: I am aware that the bindings are redundant in this case. All the same, having the same RDF document with the profile loaded to an N3 store works as expected.

____
#### Environment:

- Node v21.2.0 on Windows
- `@comunica/query-sparql-solid` v3.0.1 AND v2.9.0 - both versions behave the same

**Comment by rubensworks** [comment]
Does the problem also occur with a SELECT query?

**Comment by karelklima** [comment]
At first I wanted to write that the problem occurs with SELECT query as well, but then I did some more digging and I found out that the **mistake was on my part**.

I was using solidcommunity.net as my Solid pod host. I have created an account there in 2018 and have apparently missed the migration from the original `solid.community` domain to `solidcommunity.net` domain. During the migration my profile card was apparently not updated, so it is essentially invalid. So much for persistent URLs :-)

My current profile is: `https://karelklima.solidcommunity.net/profile/card#me`
But the IRIs in the document are: `https://karelklima.solid.community/profile/card#me`

I have created a new testing pod - everything works there as expected. Sorry for the confusion, closing this issue.
